### ISSUE TITLE
Support for login with Apple and Facebook 

### ISSUE THREAD

**Issue by ghost** [benawad/dogehouse#714]

**Comment by Mvrs** [comment]
If anyone isn't tackling this I would like to give it a go 😎

**Comment by ghost** [comment]
Yeah..... .We can authenticate userprofile not only with Github but apple, google facebook also

**Comment by benawad** [comment]
@Mvrs wait on this, it's a low priority right now

**Comment by Mvrs** [comment]
@Krishna-Ramasimha so you're saying every other has been implemented except for Apple, Facebook, and Google? Correct?

**Comment by ghost** [comment]
Like... adding support for login with these 3 and many more instead of just github and twitter cuz many might not have these accounts

**Comment by Mvrs** [comment]
ah~~~ok

**Comment by DGKSK8LIFE** [comment]
Yeah maybe try limiting/streamlining Oauth services you allow. It could become a security liability in the future if you are supporting oauth authentication/integration through 5 services at once.

**Comment by Mvrs** [comment]
@DGKSK8LIFE yeah, I agree. I also believe the design would change as well so that would require the login component to have a new design. Which would require a new Storybook component.

**Comment by AlessandroSpallina** [comment]
+1 for google authentication, non-developer guys don't have github and twitter is not popular in countries like italy
